homeassistant/components/jellyfin/config_flow.py

```diff
@@ -39,7 +45,10 @@ async def async_step_user(
         errors: dict[str, str] = {}
 
         if user_input is not None:
-            client = create_client()
+            if CONF_DEVICE_ID not in user_input:
```

Comment:
Given that CONF_DEVICE_ID is not in the USER_DATA_SCHEMA, this will never be available at this point I believe, so this check can also be removed.

Reply:
I need to write a test but I'm thinking about when you have errors and data gets repaired though next attempt, I'll have to see if that works way I believe

Reply:
OK I ended up changing the logic a bit to use a class attribute vs hijacking it on the user_input.